content/en/standards/reproducibility/_index.md

```diff
@@ -12,4 +12,68 @@ weight: 2
+
+> Reproducibility means obtaining consistent computational results using the same input data, computational steps, methods, code, and conditions of analysis.  Replicability means obtaining consistent results across studies aimed at answering the same scientific question, each of which has obtained its own data.  
+
+Reproducibility is a cornerstone of scientific research, especially for transparent evaluation of research claims. Equally important is the potential for scientific procedures to be reused and modified in order to carry out new research. While clear documentation is essential to reusability, new technologies also offer the potential to increase the reproducibility and reusability of model code. A critical metric of reproducibility is whether a model can consistently reproduce the published results that claim to have been generated by the model. Reproducibility and reusability can also be enhanced if model code is accompanied by explicit workflows that demonstrate how the model generates and transforms its output data into its published findings.
+
+
+### Minimal Standards 
+
+This set of minimal standards can be easily adopted by journals to ensure that submitted publications meet baseline reproducibility and reusability requirements. The high level goal is for an independent reviewer to be able to easily download, build, and execute a computational model and verify that it meets the reproducibility claims stated by the model author(s).
+
+Computational models must:
+
+- meet OMF minimal standards for <a href='{{< relref "/standards/accessibility" >}}'>Accessibility</a> and <a href='{{< relref "/standards/documentation" >}}'>Documentation</a>
+- include a minimal set of metadata describing the model including: name, version, description, authors and contributors, open source software license, a permanent link to a versioned codebase landing page that broadly follows the [FORCE11 software citation guidelines](https://www.force11.org/software-citation-principles)
+- include provenance of all input datasets: permanent links to versioned input data dependencies (and outputs where possible)
+- favor open file formats for data inputs and outputs (e.g., CSV, netCDF, YAML/JSON)
+- provide information on all versioned software and system dependencies (operating system, software and system libraries) with relevant software citations for key dependencies
+- include instructions on how to set up a compute environment to compile / interpret / execute the software
+  - software with large compute or data requirements can be problematic:
+    - provide representative input data samples along with sampling methodology
+    - provide durable link to wholetale or other provenance-tracked computation
+- clearly describe what reproducibility measures or metrics are applicable to the given software
+- provide range of acceptable outcomes for models with stochastic components
```

Comment:
I said "models" out of habit.  We should probably word this as "software" instead